libp2p/RLPxHandshake.cpp

```diff
@@ -32,20 +32,22 @@ RLPXHandshake::RLPXHandshake(
     m_socket(_socket),
     m_idleTimer(m_socket->ref().get_io_service())
 {
+    auto const prefixAttr = boost::log::attributes::constant<std::string>{ connectionDirectionString() };
     m_logger.add_attribute(
-        "Prefix", boost::log::attributes::constant<std::string>(connectionDirectionString()));
+        "Prefix", prefixAttr);
```

Comment:
wrapping is odd, did you run clang-format here?

Reply:
I think so but I can manually reformat this

Reply:
Better leave it like clang-format wishes, but it seems it shouldn't have done this here - the line above is 4 times longer, but only this one is wrapped

Reply:
I manually reformatted it and ran it through clang-format again and it looks like this now:

```
 auto const prefixAttr =
        boost::log::attributes::constant<std::string>{connectionDirectionString()};
    m_logger.add_attribute("Prefix", prefixAttr);
    m_errorLogger.add_attribute("Prefix", prefixAttr);
```

So I suspect I forgot to run clang-format yesterday.